setup.py

```diff
@@ -28,7 +28,7 @@
     },
     install_requires=[
         "OpenTimelineIO >= 0.12.0",
-        "openassetio == 1.0.0a14",
+        "openassetio == 1.0.0b1.rev0",
```

Comment:
[consider] We always wanted to avoid hard pins, and use semver (not sure how white space works here)
```suggestion
        "openassetio == 1.0.0b1.rev0, < 1.0.0b2.rev0",
```

Reply:
Updated as suggested in 300fa844